### PR TITLE
Update README usage to use try catchs

### PR DESCRIPTION
**Pull request by jwilander**
#### Summary
Update README usage to use try catchs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,8 +38,22 @@ import {Client4} from 'mattermost-redux/client';
 Client4.setUrl('https://your-mattermost-url.com');
 
 async function loginAndGetUser(username, password) {
-    await Client4.login(username, password);
-    return await Client4.getMe();
+    try {
+        await Client4.login(username, password);
+    } catch (error) {
+        console.error(error);
+        return null;
```
Comment (enahum) [inline README.md]: This should be returning {error}

File: README.md [hunk 2]
```diff
@@ -38,8 +38,22 @@ import {Client4} from 'mattermost-redux/client';
...
+
+    let user;
+    try {
+        user = await Client4.getMe();
+    } catch (error) {
+        console.error(error);
+        return null;
+    }
+
+    return user;
```
Comment (enahum) [inline README.md]: And this should be {data: user} as we talked

Comment (jwilander) [reply README.md]: These aren't actions?

Comment (enahum) [reply README.md]: Oh is just using the client...

Can you add another example using promises instead of async/await?

Comment (jwilander) [reply README.md]: Yeah sure

**Pull request merged**